src/editor/config/config.ts

```diff
@@ -1,4 +1,199 @@
-export default {
+export interface EditorConfig {
+
+  /**
+   * The html target element.
+   */
+  container?: string;
+
+  plugins?: any[];
```

Comment:
I think we can have better typing here for plugins.

Reply:
This seems like a partial solution until plugins gets typed.